docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,619 @@
+= Migration Tutorial: From APPUiO Public to APPUiO Cloud
+
+This document provides an overview of all the tasks required to move your applications from APPUiO Cloud to APPUiO Public.
+
+== Main Differences
+
+Before starting the migration of your applications from APPUiO Public to APPUiO Cloud, please check the xref:explanation/differences-to-public.adoc[document explaining all the major differences] between both platforms.
+
+== DNS
+
+Customers transferring their domains from APPUiO Public to APPUiO Cloud should be aware of the following:
+
+. Lowering your TTLs (Time To Live) 24 hours ahead of time will reduce the amount of time that your change takes to propagate. You can limit downtime by lowering that value, so the caching servers will check back more frequently. Typically, 60 seconds (1 minute) is a good idea.
++
+IMPORTANT: Remember to set your TTL back to the previous value after the migration.
+
+. There are different `CNAME` resource records to point to depending on which {zone} you migrated the application to; please refer to xref:zones.adoc[the {zone}s] reference document, where the corresponding `CNAME` entry is specified.
+
+== Let's Encrypt
+
+In APPUiO Cloud only `Ingress` objects can request certificates.
+
+IMPORTANT: It's not possible to request certificates in `Route` objects.
+
+The manifest below specifies an `Ingress` object with such requirement:
+
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true" # <1>
+  name: ${DEPLOYMENT_NAME}
+  namespace: oauth-test
+spec:
+  rules:
+  - host: ${ROUTE_HOST}
+    http:
+      paths:
+      - backend:
+          service:
+            name: nginx
+            port:
+              number: 80
+        path: /
+        pathType: ImplementationSpecific
+  tls:
+  - hosts:
+    - ${ROUTE_HOST}
+    secretName: ${ROUTE_HOST}
+--
+<1> Annotation providing automatic provisioning of certificates on APPUiO Cloud.
+
+You can find more information in the xref:how-to/getting-a-certificate.adoc[Getting a Certificate through Let’s Encrypt] page.
+
+For reference, it was possible to configure TLS certificates in `Route` objects in APPUiO Public, using a manifest similar to the following. This is no longer valid in APPUiO Cloud.
```

Comment:
```suggestion
For reference, it was possible to configure TLS certificates in `Route` objects in APPUiO Public, using a manifest similar to the following. 
This is no longer valid in APPUiO Cloud.
```

Reply:
I do not understand what changed here.

Reply:
As  @corvus-ch already explained: We follow the one-line-per-sentence style guide in asciidoc. So I just moved `This is no longer valid in APPUiO Cloud` to the next line.

Reply:
We are not doing such a thing anywhere else; where is such a rule documented?

Reply:
ttps://syn.tools/syn/references/style-guide.html#_asciidoc_style. Yes, I understand that that the scope of that guide and this repository do not nessesarily overlap. Hower, we from Tarazed are used to follow it and it became the defacto standard also for all we do.